provider/terraform/common~compile.yaml

```diff
@@ -14,3 +14,27 @@
 # the final module tree structure:
 <% dir = _version_name == 'beta' ? 'google-beta' : 'google' -%>
 '<%= dir -%>/provider_<%= api.prefix[1..-1] -%>_gen.go': 'templates/terraform/provider_gen.erb'
+
+<%  Dir["provider/terraform/tests/*.go.erb"].each do |file_path|
+    fname = file_path.split('/')[-1]
+-%>
+'<%= dir -%>/<%= fname.split(".erb")[0] -%>': 'provider/terraform/tests/<%= fname -%>'
+<% end -%>
+<%
+  Dir["provider/terraform/resources/*.go.erb"].each do |file_path|
+    fname = file_path.split('/')[-1]
+-%>
+'<%= dir -%>/<%= fname.split(".erb")[0] -%>': 'provider/terraform/resources/<%= fname -%>'
+<% end -%>
+<%
+  Dir["provider/terraform/data_sources/*.go.erb"].each do |file_path|
```

Comment:
Yup. We talked about this a bit in the meeting today, but Emily had realized that we don't have something generate on every api prefix that we would want, so even if we do a `compiler -a`, we'll still miss some things. Rather than trying to do something super complicated, we'll just compile/copy everything every time until it becomes enough of a nuisance that someone comes up with a good solution :)